requirements.txt

```diff
@@ -124,6 +124,8 @@ pyflakes==0.8.1
 pyhs2==0.6.0
 # sha256: TzJCJuHuhaxBUgV5up3nAylAEmAuU2IX_Q50VuFH2IY
 sasl==0.1.3
+# sha256: 6nj05rvk7z_4OHM6mUIsl7GjE2plb4N3PqnJWaSRRlw
+https://github.com/rhelmer/pyinotify/archive/9ff352f.zip#egg=pyinotify
```

Comment:
Please add a comment above the sha that points to https://github.com/seb-m/pyinotify/pull/92 and that says that we can start using `pyinotify>0.9.5` once that PR has been merged and a new release made.